.github/workflows/generate-changelogs.yml

```diff
@@ -0,0 +1,45 @@
+name: Generate Changelogs
+run-name: 'chore(release): generate changelogs for ${{ inputs.previous_release_tag }}..${{ inputs.future_release_tag }}'
+
+on:
+  workflow_dispatch:
+    inputs:
+      previous_release_tag:
+        required: true
+        description: previous release tag
+      future_release_tag:
+        required: true
+        description: future release tag
+
+jobs:
+  build:
+    name: Generate changelogs
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Generate release changelogs
+        uses: daeuniverse/changelogs-generator-action@test
+        id: changelog
+        with:
+          # https://github.com/daeuniverse/changelogs-generator-action
+          previousRelease: ${{ inputs.previous_release_tag }}
+          futureRelease: ${{ inputs.future_release_tag }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Print outputs
+        shell: bash
+        run: |
+          echo "${{ steps.changelog.outputs.changelogs }}"
+
+      - name: Create an issue with proposed changelogs
+        uses: dacbd/create-issue-action@main
+        with:
+          token: ${{ github.token }}
```

Comment:
This github.token is derived from github context, which will be the github account who triggers this workflow run, is this expected behavior or you want to use daebot as the triggerer.

Reply:
Good catch, I will update this and have it used `${{ secrets.GH_TOKEN }} instead.